buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -955,6 +955,7 @@ class BeamModulePlugin implements Plugin<Project> {
 
       project.dependencies {
         errorprone("com.google.errorprone:error_prone_core:2.3.1")
+        errorprone("jp.skypencil.errorprone.slf4j:errorprone-slf4j:0.1.2")
```

Comment:
Noting that this plugin has almost no development, mostly automated commits, and not a ton of usage. Seems fine and healthy but being aware that we might end up doing some maintenance on it if it breaks. ASL 2.0